db/migrate/20220811152628_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+
```

Comment:
[Optional] You could make the `name` required for the db too by:
3|       `t.string :name` -> `t.string :name, null: false`

Reply:
Updated `t.string :name` -> `t.string :name, null: false`